### DISCUSSION TITLE
Only track files that are actually bigger than 100MB

### DISCUSSION THREAD

**Discussion by Ljpton** [git-lfs/git-lfs#5704] [Q&A] [answered]
Hi,

I am new working with Git LFS and am a bit confused about the provided storage.

It says that you get 1GB storage for LFS assets for free, but in my understanding this would mean you can only store a maximum of, for example, 10 versions of a 100MB file, or 1 version of 10 100MB files - which isn't really helping when you try to work with several larger assets.

The other point is, that the above mentioned scenarios wouldn't even work since LFS does not only upload the files that actually are larger than 100MB, but everything which is tracked. For my project it is not practical to manually track every single file which exceeds GitHubs upload limit, and thus LFS storage will be crammed with lots of small files.

Maybe I am missing something? Or is it really just a minimal 'demo' and for most cases you will need to buy extra space? How do you handle it?

**Comment by chrisd8088** [comment, marked as answer]
It sounds like you're planning to use GitHub as your storage host, so you should probably check with their Support team if you have specific questions.  (This project is just for the open-source Git LFS client.)

That said, there's a long-standing feature request for the Git LFS client to track files above a given limit; see #282 for details.  Unfortunately, full support of this feature would require changes in the upstream Git project as well, if it's possible at all, and that work has not been undertaken.

So for the moment you have to either track files individually, by folder, by file extension, or using some other kind of path-based pattern.

You might also look at a recent suggestion from https://github.com/git-lfs/git-lfs/issues/282#issuecomment-1920472206; I haven't tried the utility mentioned there, but perhaps it will help for your use case.

**Reply by Ljpton** [reply]
Ah, I see. Then I will ask GitHub support directly and try the workaround for the large file tracking - thank you :)
